launcher/sdw_util/Util.py

```diff
@@ -108,3 +111,55 @@ def is_conflicting_process_running(list):
             sdlog.error("Conflicting process '{}' is currently running.".format(name))
             return True
     return False
+
+
+def get_qubes_version():
+    """
+    Helper function for checking the Qubes version. Returns None if not on Qubes.
+    """
+    is_qubes = False
+    version = None
+    try:
+        with open(OS_RELEASE_FILE) as f:
+            for line in f:
+                try:
+                    key, value = line.rstrip().split("=")
+                except ValueError:
+                    continue
```

Comment:
pytest is telling me that these two lines don't have test coverage

Reply:
I'll add a `bad-file-format` fixture for this case.

Reply:
Done in fd6b82456b02dc48799bb492f14ada3cf5a9c89c